Samples/Windowing/cs-winforms-unpackaged/AppForm.cs

```diff
@@ -45,6 +46,9 @@ private void InitializeComponent()
             this.MyTitleBar = new System.Windows.Forms.FlowLayoutPanel();
             this.MyWindowIcon = new System.Windows.Forms.PictureBox();
             this.MyWindowTitle = new System.Windows.Forms.Label();
+            this.clientResizeButton = new System.Windows.Forms.Button();
+            this.standardHeight = new System.Windows.Forms.Button();
+            this.tallHeight = new System.Windows.Forms.Button();
```

Comment:
Same question as for the WinUI-C# sample: any reason why this is two buttons instead of a toggle switch?